src/components/resources/ResourcesOverview.js

```diff
@@ -1,12 +1,22 @@
 import styled from 'styled-components';
 import {Resource} from './Resource';
+import {nanoid} from '@reduxjs/toolkit';
+import {useSelector} from 'react-redux';
+
+export const ResourcesOverview = () => {
+  const currentUserResources = useSelector(state => state.currentUserResources);
+
+  let currentRess = [];
+
+  Object.keys(currentUserResources).forEach(key => {
+    currentRess.push({name: key, value: currentUserResources[key]});
+  });
```

Comment:
Adding nanoid here so that the key won't change with every rerender.
```suggestion
  Object.keys(currentUserResources).forEach(key => {
    currentRess.push({name: key, value: currentUserResources[key], id: nanoid()});
  });
```

Reply:
done